### PR TITLE
QA pr

### PR DESCRIPTION
**Pull request by Volvo-hackangel**

### CODE REVIEW

File: script/QA/QA_02.txt [hunk 1]
```diff
@@ -97,7 +97,7 @@ We'll explain those cases when the time comes.
 [sha:11100ece80f4d0c7dd8198ad8798434dc5531f1a]{
 -- Page 39, Offset 42840.
 -- 今回はよくある『即死系デッドエンド』です。
- This time,%{n}we have the common 'instant death dead end'.
+ This time,%{n}we have the common "instant death dead end".
```
Comment (rschlaikjer) [inline script/QA/QA_02.txt:100]: Should still be single quotes I think?

File: script/QA/QA_02.txt [hunk 2]
```diff
@@ -138,7 +138,7 @@ He could have escaped if he had been a cat.%{n}
 [sha:555ce363b009814e84e82c1dd139deb2627cbcbf]{
 -- Page 44, Offset 42848.
 -- ？　それは身が軽い、という意味ですか？
-Hmm? When you say 'fly', what do you mean?%{n}
+Hmm? When you say "fly", what do you mean?%{n}
```
Comment (rschlaikjer) [inline script/QA/QA_02.txt:141]: Single quotes here too?

File: script/QA/QA_04.txt [hunk 3]
```diff
@@ -195,7 +195,7 @@ I'll buy you a round of S-rank sushi next%{n}time.
 -- Page 87, Offset 42926.
 -- <鰈|かれい>とか、お好きでしょう、オタク？
 // Otaku here is actually used in its original meaning, as a pronoun. ruby maybe? also recheck the pun. Newline for v-center.
-%{n}You do like <'karei'|r i g h t e y e   f l o u n d e r>, right, otaku?
+%{n}You do like <"karei"|r i g h t e y e   f l o u n d e r>, right, otaku?
```
Comment (rschlaikjer) [inline script/QA/QA_04.txt:198]: Single quotes?

File: script/QA/QA_07.txt [hunk 4]
```diff
@@ -81,7 +81,7 @@ If you know which choice I'm referring to, then try%{n}
 [sha:12dfe0774a6fcbed579487a5d7259406246132a5]{
 -- Page 144, Offset 43029.
 -- 誠実に対応してみるとよいでしょう。
-going back and being more sincere.
+going back and be more sincere.
```
Comment (rschlaikjer) [inline script/QA/QA_07.txt:84]: Revert. 
> Try going back and _being_ more sincere

File: script/QA/QA_08.txt [hunk 5]
```diff
@@ -23,7 +23,7 @@ mysteries of human history head-on,%{n}
 [sha:980e14c245f584fcd0aa28da6d05aa20b121ad63]{
 -- Page 158, Offset 43052.
 -- 真っ向から挑む知能番組『教えて！　シエル先生』、
-'Teach Me! Ciel-Sensei'―――
+"Teach Me, Ciel-Sensei!"―――
```
Comment (rschlaikjer) [inline script/QA/QA_08.txt:26]: Gonna stop calling these out but what's with the "

File: script/QA/QA_09.txt [hunk 6]
```diff
@@ -188,6 +189,7 @@ In our case, it was simply a pure, platonic%{n}conversation
 [sha:4d9019bb07c8396d980d7d9198cfa4ae139a532a]{
 -- Page 197, Offset 43126.
 -- 清らかな男女交際トークです。
+// is junior and senior ok here?
```
Comment (rschlaikjer) [inline script/QA/QA_09.txt:192]: sure

File: script/QA/QA_02.txt (hunk 1 above)
Comment (Volvo-hackangel) [reply script/QA/QA_02.txt:100]: I thought it's supposed to be double quotations and @Avuxo agreed so which is it?

Comment (Avuxo) [reply script/QA/QA_02.txt:100]: although these are in-text quotes, they're specifically _not_ wrapped in any form of quotation. That's why I said double quotes. If Ross thinks we should take the implicit quotes as a reason to use single quotes in QA that's fine.

**Review by Avuxo — APPROVED** [review]

**Review by yk1512 — APPROVED** [review]

File: script/QA/QA_02.txt (hunk 1 above)
Comment (rschlaikjer) [reply script/QA/QA_02.txt:100]: Yeah I feel like it's standard to use single quotes for non-speech 'things' like titles or terminology

Comment (Avuxo) [reply script/QA/QA_02.txt:100]: I'm fine to have it be reverted

**Pull request merged**